build

```diff
@@ -92,7 +94,10 @@ def create_package(pkgname):
         out = '{}.tar.gz'.format(pkgname)
         rmfile(out)
         info("Creating {} ...".format(out))
-        run(['tar', '-c', '-z', '-f', out, '--owner=%s' % USER, '--group=%s' % GROUP, pkgname], cwd=Globals.finalroot)
+        if platform.system() == "Darwin":
+            run(['tar', '-c', '-z', '-f', out, pkgname], cwd=Globals.finalroot)
```

Comment:
As discussed in email my preference would be to disallow creation on MacOS, since it doesn't properly support the functionality needed to assign user/group to the published tarball.

Reply:
@madscientist I've added a simple check to prevent the build script to run on macOS.